### PR TITLE
Add wrapper object as a param to the callback property

### PR DESCRIPTION
**Pull request by snirs90**
Now the callback property function get the wrapper object to it can be used inside the callback.

### CODE REVIEW

File: plugins/restful/RestfulEntityBase.php [hunk 1]
```diff
@@ -583,7 +583,7 @@ public function viewEntity($entity_id, $request, stdClass $account) {
           throw new Exception(format_string('Process callback function: @callback does not exists.', array('@callback' => $callback_name)));
         }
 
-        $value = call_user_func($info['callback']);
+        $value = call_user_func($info['callback'], $wrapper);
```
Comment (amitaibu) [inline plugins/restful/RestfulEntityBase.php:588]: Add the info that we pass the wrapper [here](https://github.com/Gizra/restful/blob/7.x-1.x/plugins/restful/RestfulEntityBase.php#L51)

Comment (snirs90) [inline plugins/restful/RestfulEntityBase.php:588]: Added

File: plugins/restful/RestfulEntityBase.php [hunk 2]
```diff
@@ -50,6 +50,8 @@
    *    (e.g. $wrapper->field_reference->label()). Defaults to FALSE.
    *  - "callback": A callable callback to get a computed value. Defaults To
    *    FALSE.
+   *    The callback function receive as first argument the node
```
Comment (amitaibu) [inline plugins/restful/RestfulEntityBase.php]: node => entity

**Comment by amitaibu** [comment]
Merged (with correct credit).

**Pull request closed without merging**